src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -600,12 +605,19 @@ public override void _Process(float delta)
                 return;
             }
 
-            if (!TransitionFinished)
-                return;
-
             OnEditorReady();
         }
 
+        // Save if wanted
+        if (TransitionFinished && wantsToSave)
+        {
+            // Auto save after editor entry is complete
```

Comment:
I think this comment should replace the one on line 611, and then there's no need to have this line at all.